micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/Landing/LandingComponent.js

```
@@ -8,7 +8,7 @@ import YoutubeVideo from "./YoutubeVideo";
 const LandingComponent = ({ config }) => {
   const { t } = useTranslation(); // To handle translations (if needed)
   const { heading, subsections, url } = config;
-  let redirectPathOtpLogin = `/${window?.contextPath}/employee`;
+  let redirectPathOtpLogin = `/${window?.contextPath}/employee/user/landing/select-role`;
```

Biome

[error] This let declares a variable that is only assigned once.

'redirectPathOtpLogin' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)